db/migrate/20160620085231_add_new_saving_fields_to_online_application.rb

```diff
@@ -0,0 +1,18 @@
+class AddNewSavingFieldsToOnlineApplication < ActiveRecord::Migration
+  def change
+    rename_column :online_applications, :threshold_exceeded, :min_threshold_exceeded
+    add_column :online_applications, :max_threshold_exceeded, :boolean
+    add_column :online_applications, :over_61, :boolean
+    add_column :online_applications, :amount, :integer
+
+    reversible do |dir|
+      dir.up do
+        migrate_sql = <<~SQL
+          UPDATE online_applications
+          SET max_threshold_exceeded = min_threshold_exceeded
```

Comment:
I'm not sure about this, because if the `min_threshold_exceeded == false`, then this sets `max_threshold_exceeded = false` whereas I assumed it would be `nil` in that case. Maybe not a big deal.